osu.Game/Beatmaps/Drawables/BeatmapSetHeader.cs

```diff
@@ -112,7 +106,7 @@ public PanelBackground(WorkingBeatmap working)
                         Depth = -1,
                         Direction = FillDirection.Horizontal,
                         RelativeSizeAxes = Axes.Both,
-                        // This makes the gradient not be perfectly horizontal, but diagonal at a ~40� angle
+                        // This makes the gradient not be perfectly horizontal, but diagonal at a ~40° angle

```

Comment:
Here seems a non-ASCII character, so I changed the file to UTF-8 BOM, otherwise Visual Studio will treat it incorrectly.